tasks/ipv6.yml

```diff
@@ -26,6 +26,12 @@
     - ipv6
     - grub
 
+- name: Does local /proc have ipv6 settings?
+  become: true
+  ansible.builtin.stat:
+    path: /proc/sys/net/ipv6
+  register: has_ipv6
```

Comment:
instead of using `register`  and repeating it later on, why not use `set_fact`?

Reply:
Moved it to facts.yml so it gets run at the start.